cupy/cuda/compiler.py

```diff
@@ -78,7 +78,12 @@ def compile_using_nvrtc(source, options=(), arch=None):
         return ptx
 
 
-def _preprocess(source, options=()):
+def _preprocess(source, options=(), arch=None):
+    if not arch:
```

Comment:
These two lines are redundant.
Cloud you remove it?

Reply:
I had noticed the lines are redundant, but I was afraid that it is not very clear the `arch` argument is required for `_preprocess` function.

I'm considering to omit the default values of both of `options` and `arch` arguments. How do you think?

Reply:
OK, Please omit.

Reply:
@okuta Removed.